backend/client/deepchecks_client/__init__.py

```diff
@@ -76,7 +76,8 @@ def get_or_create_model(
         alerts_delay_labels_ratio: float = 1.0,
         alerts_delay_seconds: int = 3600 * 72,  # 3 days
         model_notes: t.Optional[t.List[t.Dict[str, str]]] = None,
-        monitoring_frequency: str = 'day'
+        monitoring_frequency: str = 'day',
+        s3_path: t.Optional[str] = None,
```

Comment:
@matanper can you replace s3_path with obj_store_path the reason is because s3 is AWS implementation of object store and in the future we will probably include support for Azure blob storage and GCP so I prefer that we will expose a generic name to user instead of vendor specific one